internal/api/schemas/v1.yaml

```diff
@@ -154,6 +154,103 @@ paths:
             application/json:
               schema:
                 "$ref": "#/components/schemas/Error"
+  "/namespaces/{namespace}/gateways/{name}/tokens":
+    post:
+      description: Creates a new bootstrap token for a gateway based off of its unique name.
+      operationId: createTokenForGatewayInNamespace
+      parameters:
+      - name: namespace
+        in: path
+        description: Namespace of gateway for token to be created
+        required: true
+        schema:
+          type: string
+      - name: name
+        in: path
+        description: Name of gateway for token to be created
+        required: true
+        schema:
+          type: string
+      responses:
+        '201':
+          description: token response
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/BootstrapToken"
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Error"
+  "/namespaces/{namespace}/gateways/{name}/tokens/{id}":
```

Comment:
Should we have a retrieval endpoint or should we assume that the token is only available on generation? I'm not sure how secure we need these to be.

Reply:
Yea, I wasn't sure about this either, thoughts @andrewstucki?

Reply:
Will keep this for now and remove later if not needed (assuming all of this may be subject to change depending on how our VM/API work evolves), agreed it might be useful for returning metadata about a token like TTL, revocation status, etc.